packages/common/driver-definitions/src/storage.ts

```diff
@@ -307,6 +307,13 @@ export interface IDocumentDeltaConnection
 	 */
 	relayServiceAgent?: string;
 
+	/**
+	 * A list of optional features that ordering service supports.
+	 * Features supported might be service specific.
+	 * If we have standardized features across all services, they need to be exposed in more structured way.
+	 */
+	supportedFeatures?: Record<string, unknown>;
```

Comment:
I'm not sure this is a good idea to expose protocol level feature set to the loader. This results in some level of leakage, where this loader layer is exposed to concepts, it should not be, and thus there is a temptation to start depending on some of them.
Feature set is supposed to be service specific, and each driver should ideally project them down to the loader in type safe manner, where applicable.